c2corg_ui/static/js/gpxupload.js

```diff
@@ -59,6 +61,60 @@ app.GpxUploadController = function($scope) {
 
 
 /**
+ *
+ * @param {Array.<ol.Feature>} features - invalid features that have to be checked
+ * @returns {Array.<ol.Feature>} features - this geometry is valid
+ * @private
+ */
+app.GpxUploadController.prototype.validatedFeatures_ = function(features) {
+  for (var i = 0; i < features.length; i++) {
+    /**
+     *
+     * @type {ol.geom.Geometry}
+     */
+    var geom = /**@type{ol.geom.Geometry}*/ (features[i].getGeometry());
+
+    /**
+     *
+     * @type {ol.geom.GeometryType}
+     */
+    var geomType = geom.getType();
+
+    // TODO validate LineString? pass now - no problem we know about?
+    // if (geomType === 'LineString') {
+    // }
+
+    if (geomType === 'MultiLineString') {
+      var multiLineString = /**@type{ol.geom.MultiLineString}*/ (geom);
+      /**
+       *
+       * @type {Array.<ol.geom.LineString>}
+       */
+      var lineStrings = multiLineString.getLineStrings();
+
+      for (var j = 0; j < lineStrings.length; j++) {
+        if (lineStrings[j].getCoordinates().length === 1) {
```

Comment:
No, because it checks only that [if geom is instance of LS or MultiLS](https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/utils.js#L306-L313) and doesn't validate anything else with coordinates. This fix should be applied only for MultiLS so I should leave it like it is.